benchmarks/freetype2-2017/Dockerfile

```diff
@@ -24,4 +24,7 @@ RUN apt-get update &&  \
     libarchive-dev \
     zlib1g-dev
 
+RUN git clone git://git.sv.nongnu.org/freetype/freetype2.git
+RUN git clone https://github.com/unicode-org/text-rendering-tests.git TRT
+
```

Comment:
Consider also moving the `git checkout <version we want>` commands right after the corresponding clone-s.

(later we might also consider doing the clone/checkout in a single RUN step with pruning the repo to save space)

Reply:
Yes let me file a bug for this, both of these can be done in one go. This one saves a whole bunch of network bandwidth resources which we are wasting atm.

Reply:
Filed https://github.com/google/fuzzbench/issues/611.